.env.example

```diff
@@ -15,3 +15,10 @@ SENTRY_DNS=
 
 # Log level to use, DEBUG by default in dev
```

Comment:
Locally we should have a ".env" file ready for local development, so you can rename this file ".env"

Reply:
ok ! 
- i deleted `.env.example`
- remove `.env` from the gitignore
- simplified the install doc